chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts

```diff
@@ -0,0 +1,26 @@
+import {Injectable} from '@angular/core';
+import {TranslateService} from '@ngx-translate/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BrowserLocaleService {
+  constructor(public translate: TranslateService) {
+  }
+
+  public findLanguage() {
+    let passedLanguage;
+    const recievedLanguage = navigator.language;
```

Comment:
Don't need to create a variable here.  Just use `navigator.language' directly in if statement.

Reply:
Removed constant and added directly to page.